### ISSUE TITLE
nvda does not read screen to choose virtual box boot disk 

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#13176]
### Steps to reproduce:
create a virtual machine using the virtual box.
I followed the settings suggested by the program, virtual machine for windows 10.
now press applications on the created virtual machine.
this one with the arrows until it starts.
press enter, press enter also under normal boot.
The virtual box will display a window asking you to choose which virtual machine boot disk, including choosing to boot from a disk image.
try navigating this screen with tab or arrows.
Due to my inexperience with the software if I choose any option I can't even make this screen appear again, I just create a new virtual machine. 
### Actual behavior:
when walking with tabe or arrows through the window to choose the boot disk nvda only reports unknown. 
### Expected behavior:
nvda should read the options in this window. 
#### NVDA installed/portable/running from source:
instaled
#### NVDA version:
nvda.exe, NVDA alpha-24329,d3444424

#### Windows version:
11 22000.376
#### Name and version of other software in use when reproducing the issue:
VirtualBox.exe, Oracle VM VirtualBox 6.1.30.148432

#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If NVDA add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes
[ovb.txt](https://github.com/nvaccess/nvda/files/7738913/ovb.txt)

**Comment by gregjozk** [comment]
Hello,

May be [this reading] (https://www.virtualbox.org/ticket/629#comment:30) might help you. please test menssioned dev version of VB and report results there. as I know and understand situation, on NVDA's side there could be done little to none.

regards

**Comment by fernando-jose-silva** [comment]
thank you very much for the feedback.
I downloaded the testing version of the virtual box and did the tests.
a solution was given, now the wizard for creating the new virtual machine is offered the possibility of indicating the iso installation of windows, no longer needing to interact with the screen located in this problem.

**Comment by josephsl** [comment]
Hi,

May 2022: based on the last comment, does this mean we can close this as Oracle resolved this?

Tahnks.
